### PR TITLE
Implements operations 0 through 8 of the revised spec.

### PR DESCRIPTION
**Pull request by alexnj**
Operations are now defined and portable across PRE_UA and ON_UA. See the updated EXPLAINER.md for supported operations.

Other changes:
 * Point to a new gist that has the revised spec.
 * Switch to Closure Compiler.

/cc @micahheineckoptimizely @smoroz

### CODE REVIEW

File: src/constants.js [hunk 1]
```diff
@@ -17,3 +17,16 @@
 export const PRE_UA = 0x1;
 export const ON_UA = 0x2;
 export const ONCE = 0x4;
+
+export const OPERATIONS = {
+  OP_CUSTOM_JS: 0,
+  OP_PREPEND: 2,
```
Comment (micahheineckoptimizely) [inline src/constants.js:23]: Was code `1` intentionally skipped?

File: src/pre-ua-transform-applicator.js [hunk 2]
```diff
@@ -14,17 +14,38 @@
  * limitations under the License.
  */
 
+import {OPERATIONS} from './constants.js';
+
+const opToCloudFlareOpMap = {
+  [OPERATIONS.OP_INSERT_BEFORE]: ['before', (html) => [html, {html: true}]],
```
Comment (micahheineckoptimizely) [inline src/pre-ua-transform-applicator.js:20]: This is very clever 😄 
I learned a few things today, thanks!

File: src/config.js [hunk 3]
```diff
@@ -16,6 +16,6 @@
 
 export default {
   getAbConfigEndpoint: (example) => {
-    return `https://gist.githubusercontent.com/alexnj/4c8d9198d16b238e4c7040250f052284/raw/${example}`;
+    return `https://gist.githubusercontent.com/alexnj/56e6a1bed280d3111e623f7f6da4d139/raw/${example}`;
```
Comment (micahheineckoptimizely) [inline src/config.js:19]: I left a comment on the new version: https://gist.github.com/alexnj/56e6a1bed280d3111e623f7f6da4d139?permalink_comment_id=4349894#gistcomment-4349894

**Review by micahheineckoptimizely — CHANGES_REQUESTED** [review]
Mostly LGTM, only potential minor changes.

File: src/constants.js (hunk 1 above)
Comment (alexnj) [reply src/constants.js:23]: We had `OP_CUSTOM_CSS` that we retired.

**Pull request merged**